Cargo.toml

```
@@ -63,7 +63,7 @@ exclude = ["e2e_test/udf/wasm", "lints"]
 resolver = "2"
 
 [workspace.package]
-version = "1.5.0-alpha"
+version = "1.6.0"
```

In my mind, the version here should be the version of the next release.

```suggestion
version = "1.7.0-alpha"
```

(Not 100% sure it's a best practice)

---

fixed, what's the meaning of `alpha`